cmd/shell.go

```diff
@@ -206,14 +209,28 @@ func DebugServer(addr string) {
 	}
 }
 
-// ReadJSONFile takes a file path as an argument and attempts to
+// ReadConfigFile takes a file path as an argument and attempts to
 // unmarshal the content of the file into a struct containing a
-// configuration of a boulder component.
-func ReadJSONFile(filename string, out interface{}) error {
+// configuration of a boulder component. If the file contains a
+// "Features" field it will try to initialize the features
```

Comment:
Rather than using special semantics for one named field, I think it would be better for any main.go that wants to use feature flags to include a `Features map[string]bool` in its config struct, then explicitly pass that to `features.Set()`. That reduces the amount of action-at-a-distance.